spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user, aliases: [:author] do
```

Comment:
```suggestion
  factory :user do
```
I think we can remove `aliases` option. What do you think about it?